File: Math/src/Max_min.java
public class Max_min {
	public static void main(String[] args) {

		int a[] = { 0, -1, -1, -1, -1 };
		int n = 4, v0, v1, t0, i;
		v1 = v0 = a[0];
		while (n > 0) {
			for (i = 0; i <= n; i++) {
				System.out.println(a[i]);
				t0 = a[i];
				if (t0 < v0)
					v0 = t0;
				else if (t0 > v1)
					v1 = t0;
			}
			break;
		}
		System.out.println("max" + v1 + "min" + v0);
	}
}

File: c++work/Max_min/src/Max_min.cpp
//============================================================================
// Name        : Max_min.cpp
// Author      : cbb
// Version     :
// Copyright   : Your copyright notice
// Description : Hello World in C++, Ansi-style
//============================================================================

#include <iostream>
#include <climits>
using namespace std;

struct max_min {
	int min,
	int max
} min_max_t;
const int MI = MAX_INTEGER;
const int Mi = MIN_INTEGER;

inline min(int a0, int a1) {return (a0 < a1 ? a0 : a1);}

inline max(int a0, int a1) {return (a1 == MI ? a0: (a0 > a1 ? a0 : a1));}

min_max_t minMax(int a1, int a2 = MI, int a3 = MI, int a4 = MI, int a5 = MI) {
	min_max_t result;
	result.min = min(a1, min(a2, min(a3, min(a4, a5))));
	result.max = max(a1, max(a2, max(a3, max(a4, a5))));
	return result;
}

int main() {
	cout << "Hello World!!!" << endl; // prints Hello World!!!
	return 0;
}
